.github/workflows/ci.yml

```diff
@@ -494,7 +494,7 @@ jobs:
           matrix.job == 'test' &&
           matrix.profile == 'release' &&
           !startsWith(github.ref, 'refs/tags/'))
-        run: target/release/deno run -A --unstable ext/websocket/autobahn/fuzzingclient.js
+        run: target/release/deno run -A --unstable --config tests/config/deno.json ext/websocket/autobahn/fuzzingclient.js
```

Comment:
Maybe we should add a deno.json in the tools directory that gets auto-discovered? Then we don't need to add `--config` here. It's also strange for these to be using something in tests/config

Reply:
Nice suggestion. I didn't know `deno.json` auto-discovery worked like that. TIL. Can we not also add a `deno.json` to `tests` or would that interfere with the tests?

Reply:
It would most likely interfere we some of the tests - but +1 for adding another `deno.json` in `tools/` directory